packages/gasket-cli/lib/bin/init.js

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env node
```

Comment:
This shouldn't be invoked directly, correct? in which case this line is unnecessary.
```suggestion
```
The `init` function is only executed by `run.js`, so could live in the lib dir (not bin) - see previous comment.